pkg/api/handlers/compat/networks.go

```diff
@@ -289,25 +289,44 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 
 func RemoveNetwork(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
-	config, err := runtime.GetConfig()
-	if err != nil {
-		utils.InternalServerError(w, err)
+	ic := abi.ContainerEngine{Libpod: runtime}
+
+	query := struct {
+		Force bool `schema:"force"`
```

Comment:
Does this parameter need document in pkg/api/server/register_networks.go?

Reply:
It was already documented, just not implemented.